stellar/deploy-contract.js

```diff
@@ -8,12 +8,77 @@ const { getDomainSeparator, getChainConfig } = require('../common');
 const { prompt, validateParameters } = require('../common/utils');
 const { weightedSignersToScVal } = require('./type-utils');
 const { ethers } = require('hardhat');
-const { readFileSync } = require('fs');
+const { writeFileSync, readFileSync } = require('fs');
+const fetch = require('node-fetch');
+const path = require('path');
 const {
     utils: { arrayify, id },
 } = ethers;
 require('./cli-utils');
 
+const AXELAR_RELEASE_BASE_URL = 'https://static.axelar.network/releases/axelar-cgp-stellar';
+
+const SUPPORTED_CONTRACTS = new Set([
+    'axelar_gateway',
```

Comment:
Similar to Sui deploy-contract.js, I'd like a mapping between the preferred name in our configs and the actual contract name. The config and CLI input should use `AxelarGateway` etc. Basically, makes it very easy to loop over any chain and get the address for the relevant axelar contract